examples/simple_regional_with_networking/outputs.tf

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "kubernetes_endpoint" {
+  sensitive = true
```

Comment:
All public outputs must have descriptions.

Reply:
I added descriptions for this example. Other examples do not have desc for some outputs like k8 endpoint, client_token ,ca_certificate. I can add them in a separate PR if needed.